packages/orbit-components/src/Accordion/AccordionSection/components/SectionHeader.tsx

```diff
@@ -54,7 +54,7 @@ const AccordionSectionHeader = ({
     <>
       <div className="flex grow items-center">{children}</div>
       {!expanded && expandable && (
-        <div className="ms-600 flex">
+        <div className={cx(isInteractive && "hidden", "ms-600 flex")}>
           {actions || (
             <Button onClick={handleButtonClick} type="secondary">
               Open
```

Comment:
That's out of the scope of this a11y fix task, but I guess the `Open` text should be replaced by a prop, because of the translation. Should I fix it within this PR or create a new task for that?

Reply:
This is a severe mistake in our end… and it has been like this for years. Which leads me to think nobody is really using it… So that's why my advice is to replace this by just a chevron icon button. And, on mobile, it does not render as a button, to avoid the nested interactive elements. WDYT?